tests/draft7/refRemote.json

```diff
@@ -167,5 +167,25 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description":"$ref to a different schema version",
+        "schema":{
+            "type" : "object",
+            "properties" : {
+                "foo" : {"type" : "string"},
+                "bar" : {"$ref" : "http://localhost:1234/integer-draft4.json"}
+            }
+        },
+        "tests":[
+            {
+                "description": "schema 7 referencing schema 4",
+                "data" : {
+                    "foo" : "a value",
+                    "bar" : 15
```

Comment:
I think we want 10 here, and validation outcome of `false`.  If an implementation applies the local meta-schema to the remote schema and validates it, then it doesn't matter what the number is- the boolean `exclusiveMinimum` will fail validation against the draft-06 meta-schema.

However, if the implementation does not actually validate against the meta schema (and implementations are not required to do so), then it's possible that an implementation might just ignore the boolean `exclusiveMinimum`, which means it would improperly accept 10 as an instance.